src/assets/locales/de/servicerelease.json

```diff
@@ -16,6 +16,13 @@
   "loadmore": "Mehr laden",
   "documentDeleteSuccess": "Document deleted successfully",
   "documentDeleteError": "Something went wrong.",
+  "global": {
+    "actions": {
+      "confirm": "Confirm",
+      "cancel": "Cancel",
+      "save": "Save"
+    }
+  },
```

Comment:
@lavanya to reduce future efforts can you please directly add the german translation. As far as i remember Martin had a tool for that - isnt it? @oyo